README.md

```diff
@@ -1,8 +1,9 @@
 camprotR
 ================
 
-
 <!-- badges: start -->
-[![Travis build status](https://travis-ci.org/CambridgeCentreForProteomics/camprotR.svg?branch=master)](https://travis-ci.org/CambridgeCentreForProteomics/camprotR)
-[![codecov](https://codecov.io/gh/CambridgeCentreForProteomics/camprotR/branch/master/graph/badge.svg)](https://codecov.io/gh/CambridgeCentreForProteomics/camprotR)
+
+[![R-CMD-check](https://github.com/CambridgeCentreForProteomics/camprotR/workflows/R-CMD-check/badge.svg)](https://github.com/CambridgeCentreForProteomics/camprotR/actions)
+[![Codecov test
+coverage](https://codecov.io/gh/CambridgeCentreForProteomics/camprotR/branch/master/graph/badge.svg)](https://codecov.io/gh/CambridgeCentreForProteomics/camprotR?branch=master)
```

Comment:
Should 'Codecov test coverage' all be on one line? Not sure if that matters?

Reply:
Yes it should be whoops!

Reply:
I assume you left that there for me to find so I could at least feel like I contributed something :wink: